src/lib/config/userconfig.go

```diff
@@ -261,3 +261,16 @@ func ScannerSkipUpdatePullTime(ctx context.Context) bool {
 func BannerMessage(ctx context.Context) string {
 	return DefaultMgr().Get(ctx, common.BannerMessage).GetString()
 }
+
+// AuditLogEnabled returns the audit log enabled setting for a specifc operation
+func AuditLogEnabled(ctx context.Context, operation string) bool {
```

Comment:
The semantics of this function don't seem very clear, judging from the function name and its internal implementation logic.

Reply:
updated